frame/utility/src/lib.rs

```diff
@@ -155,12 +139,8 @@ decl_module! {
 			Pays::Yes,
 		)]
 		fn batch(origin, calls: Vec<<T as Trait>::Call>) {
-			let is_root = ensure_root(origin.clone()).is_ok();
 			for (index, call) in calls.into_iter().enumerate() {
-				if !is_root && !T::IsCallable::filter(&call) {
-					Self::deposit_event(Event::Uncallable(index as u32));
-					return Ok(())
-				}
+				// TODO TODO: root no longer bypass IsCallable filter.
```

Comment:
`dispatch_bypass_filter `?

Reply:
ah I removed the TODO and put this into breaking change in this PR description.
dispatch_bypass_filter is not available here because this is the outer call not a pallet call.

The outer call for now only implements Dispatchable. and there is no way to bypass the filter. This was on design in order not to bypass BaseCallFilter unintentionnally.
I can fix it by introducing a new trait on the outer call that allow to bypass filter. or by introducing a way to build an origin withut BaseCallFilter.
But anyway I'm not sure of the real need actually

Reply:
I see, I am not familiar with the requirements of utility::batch, but to me your decision looks better; i.e. _there is no easy way to bypass the filter_.